### DISCUSSION TITLE
[Projects Beta] How to make new project public?

### DISCUSSION THREAD

**Discussion by Relequestual** [community/community#6367] [Projects and Issues] [answered]
It looks like it's not currently possible to make a new Beta style Project public. Is this correct?

In the access settings, it mentions that projects can be public.

I guess this is not yet implemented for new Beta Projects?

(Great work though 😅)

**Comment by peppy** [comment]
Any chance of an answer to this? As an open source organisation, it's basically impossible for us to test this feature if it can't be made public.

**Reply by digisomni** [reply]
Yeah, is that an oversight?

**Reply by timoyates** [reply]
Hi folks - this feature will be enabled for folks in the beta very soon 😊

**Reply by Relequestual** [reply]
@timoyates want to do this as a main reply so it can be marked as answere, or waiting for release so there's a visible solution?

**Comment by FrankHesping** [comment]
Same here: I want to make our project board visible to GitHub User outside my organization. For now, I can only add people tp the project which are part of the org.

**Comment by timoyates** [comment, marked as answer]
Hi folks - this feature will be enabled for folks in the beta very soon 😊

**Reply by Relequestual** [reply]
Curious, has this work been scheduled for completion/release yet?

**Reply by peppy** [reply]
@Relequestual as of recent, it's already available for me in the settings.

**Reply by timoyates** [reply]
Correct - this was turned on shortly after Universe (October 28)
